src/app/Main/Body/QueryPage/Results/TableResults/Results-TableResults.component.tsx

```diff
@@ -17,16 +17,16 @@ const tableDivStyle = css`
     flex-direction: row;
 `;
 
-interface TestInterface {
+interface TableResultsInterface {

```

Comment:
reject: should this be TableResultesProps?

Reply:
Yes, it was changed in the commit "Unify names of interfaces".